api_tests/nodes/filters/test_filters.py

```diff
@@ -142,6 +142,13 @@ def test_non_mutating_list_filtering_tests(
         assert child_node_two._id not in ids
         assert project._id in ids
 
+    #   test_root_ne_with_title_id_excludes_children_with_query_in_title_or_id
+        id_or_title = 'filter[title,id]={}'.format(child_node_two._id)
+        url = '{}{}&{}'.format(root_ne_url, parent_project._id, id_or_title)
+        res = app.get(url, auth=user.auth)
+        assert res.status_code == 200
+        assert len(res.json['data']) == 0
```

Comment:
This would be a better test if it searched for exactly something. Preferably it should turn up two or three nodes, one that matches on the title condition and one that matches on the id condition and possibly one that matches on both at the same time, but definitely excludes something.

So, for example, if you had two node structures:

- id: AAAA name: Root 1
    - id: AAAB name: Friend of CCCC (this is a child of AAAA)
- id: CCCC name: CCCC
    - id: CCCD name: Child of CCCC (this is a child of CCCC)
- id: EEEE name: Not related to CCCC

So if you were filtering title,id = CCCC and the ne root = AAAA, You should get back 
- CCCC
- CCCD
- EEEE

But not AAAB and definitely not AAAA. Does that make sense?

Reply:
Nice!! I have addressed this in the PR update.